-Test 1: Teste 1 (-10.000)
Incorrect program result
 --- Input ---
> 123.56

 --- Program output ---
>Digite um valor a ser sacado: Traceback (most recent call last):
>  File "atm.py", line 19, in <module>
>    if resto3==0:
>NameError: name 'resto3' is not defined

 --- Expected output (numbers)---
>6
>0
>0
>1
>1

-Summary of tests
>+------------------------------+
>|  1  test run/ 0 tests passed |
>+------------------------------+

